charts/gitops-server/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  # FIXME check the app name
+  repository: ghcr.io/weaveworks/wego-app
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.6.2"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+logLevel: info
+# Additional arguments to pass in
+# additionalArgs: []
+
+# To enable enterprise OIDC:
+# envVars:
+# - name: WEAVE_GITOPS_AUTH_ENABLED
+#   value: true
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
+  # Specifies whether the clusterRole & binding to the service account should be created
+  create: true
+  # List of resources that the service account can impersonate
+  impersonationResourceNames: []
+  # Limit the type of principal that can be impersonated
+  # impersonationResources: [ "users", "groups" ]
+  # Specify whether additional permissions should be granted to the service account
+  # in order to work with weave-gitops enterprise
+  viewSecrets: []
+
+# This should not be used in production
+testUser:
+  # Whether the wego-admin user should be created.
+  # If you use this make sure you add it to rbac.impersonationResourceNames.
+  create: false
+  # helm will base64 encode these
+  username: not-production-worthy
+  password: seriously-change-me
```

Comment:
we should note that these need to be bcrypted not plain-text

Reply:
is that in addition to base64 encoded to make them k8s secrets?

Reply:
yes. the code will do this: https://github.com/weaveworks/weave-gitops/blob/main/pkg/server/auth/server.go#L308-L313

Reply:
(turns out helm has a handy [htpasswd function](https://helm.sh/docs/chart_template_guide/function_list/#htpasswd)